### PR TITLE
GEOMESA-1264 Change csv export to have with-types flag

### PR DESCRIPTION
**Pull request by cfkelly**
* Adds a --without-types parameter to be used with delimited text formats (csv|tsv) to omit type headers on export.

Signed-off-by: Charles Kelly <charles.kelly@ccri.com>

### CODE REVIEW

File: geomesa-tools/README.md [hunk 1]
```diff
@@ -766,6 +766,9 @@ To export features, use the `export` command.
            Accumulo user name
         --visibilities
            Accumulo scan visibilities
+        --without-types
```
Comment (elahrvivaz) [inline geomesa-tools/README.md]: What do people think about making this `--no-header` instead?

Comment (cfkelly) [inline geomesa-tools/README.md]: That would be more intuitive. I'll get that fixed.

**Comment by elahrvivaz** [comment]
looks good to me. @jahhulbert-ccri can you take a peek?

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/FeatureExporter.scala [hunk 2]
```diff
@@ -122,7 +122,7 @@ object ShapefileExport {
   }
 }
 
-class DelimitedExport(writer: Writer, format: Formats) extends FeatureExporter with LazyLogging {
+class DelimitedExport(writer: Writer, format: Formats, withTypes: Boolean = true) extends FeatureExporter with LazyLogging {
```
Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/FeatureExporter.scala]: picky - rename variable to ```withHeader```

**Comment by jahhulbert-ccri** [comment]
one picky comment...then ok.

**Comment by jahhulbert-ccri** [comment]
i'll test and merge

**Comment by jahhulbert-ccri** [comment]
actually needs readme stuff

**Pull request merged**